src/objects/Mesh.js

```diff
@@ -160,7 +160,7 @@ THREE.Mesh.prototype.raycast = ( function () {
 							distance: distance,
 							point: intersectionPoint,
 							face: new THREE.Face3( a, b, c, THREE.Triangle.normal( vA, vB, vC ) ),
-							faceIndex: null,
+							faceIndex: i/3,
```

Comment:
For indexed `BufferGeometry` when offsets are present, how is the quantity `i / 3` is a useful value? How do you use that value?